deploy-vm.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
Let's move the deploy-vm.sh script to `scripts/ec2_deployment/`

Reply:
Also, can you please change the name of the script to `deploy.sh` only after moving.